packages/fxa-settings/src/pages/Signin/utils.ts

```diff
@@ -188,9 +199,24 @@ const getNavigationTarget = async ({
   return { to: '/settings' };
 };
 
-export const handleGQLError = (error: any) => {
+const handleAuthUiError = (error: any): { error: BeginSigninError } => {
+  const { errno } = error as BeginSigninError;
+  if (errno && AuthUiErrorNos[errno]) {
+    return { error };
+  }
+  return { error: AuthUiErrors.UNEXPECTED_ERROR as BeginSigninError };
+};
+
+export const getHandledError = (error: any) => {
```

Comment:
BTW, there is an annoying type discrepancy between `AuthUiError` and `AuthError`. Even if I wanted to move this to a shared location for use across container components instead of just signin, we'd need to reconcile optional vs required values.